common/djangoapps/student/admin.py

```diff
@@ -238,37 +261,40 @@ def get_search_results(self, request, queryset, search_term):
     def queryset(self, request):
         return super(CourseEnrollmentAdmin, self).queryset(request).select_related('user')
 
-    def has_permission(self, request, method):
```

Comment:
**Note to reviewers:** I updated `CourseEnrollmentAdmin` to be consistent with `LoginFailuresAdmin` in how it handles feature-enablement checking and forwards to `super` for each permission check. Not only does this create consistency in the code (for readability), but it fixes an issue where each permission check method has a different method signature.

You can see the extracted `_Check` class above that is now commonly used by both Admin views.